client/lib/signup/step-actions.js

```diff
@@ -41,7 +41,7 @@ function createCart( callback, dependencies, data ) {
 			themeItem
 		};
 
-		SignupCart.addToCart( cartKey, [ domainItem ], error => callback( error, providedDependencies ) );
+		addItems( [ domainItem ], 'signup', error => callback( error, providedDependencies ) );
```

Comment:
This will break PayPalExpress. Should use `cartKey` here, or figure how to do it in the backend.